### DISCUSSION TITLE
Is there any way we can use adMob for Blazor Hybrid ?

### DISCUSSION THREAD

**Discussion by itkunalce** [dotnet/aspnetcore#58921] [Q&A]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

Is there any way we can use adMob for Blazor Hybrid ??
If you add sample for Blazor Hybrid?

### Describe the solution you'd like

If you provide me some solution where my existing adsense replace with adMob. or when any page change then we can start reward ads.

**Comment by halter73** [comment]
It's unlikely that we'll produce a sample for Google AdMob, but we welcome anyone who wants to post a sample in the discussion.

**Comment by itkunalce** [comment]
I think I need to change question. How can we place .net MAUI Component In Blazor Component ?
